tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
The problem with passing a parameter like `'--plugin-option=tmpfs:keep_mounted=True'` is that argparse will think its a parameter it should be parsing and will raise an exception. By bundling the parameters in a single string, it correctly identifies it as a parameter value. I think another option that works would be adding a space before the parameter: `' --plugin-option=tmpfs:keep_mounted=True'`